app/server/src/controllers/appsController.ts

```diff
@@ -3,13 +3,18 @@ import { AppLocals } from "../types";
 
 export class AppsController {
     static getApp = (req: Request, res: Response) => {
-        const { configReader, appsPath } = req.app.locals as AppLocals;
+        const {
+            configReader, appsPath, wodinConfig, wodinVersion
+        } = req.app.locals as AppLocals;
         const { appName } = req.params;
         const config = configReader.readConfigFile(appsPath, `${appName}.config.json`) as any;
         if (config) {
             const view = `${config.appType}-app`;
             // TODO: validate config against schema for app type
-            res.render(view, { appName, title: config.title });
+            const viewOptions = {
+                appName, appTitle: config.title, courseTitle: wodinConfig.courseTitle, wodinVersion
+            };
+            res.render(view, viewOptions);
         } else {
             res.status(404).render("app-not-found", { appName });
```

Comment:
```suggestion
            res.status(404).render("Not Found", { appName });
```

I think it's about time to introduce standard error handling class. We should also log errors, since we now have logger implementation.

Reply:
Good idea!